choco/test.ps1

```diff
@@ -0,0 +1,57 @@
+param(
+  [string]$cpu
+)
+
+if (!$cpu) {
+  $cpu = "x64"
+}
+if ($cpu -eq "x86") {
+  $options = "-forcex86"
+}
+
+"Running tests for $cpu"
+$ErrorActionPreference = "Stop"
+
+if ($env:APPVEYOR_BUILD_VERSION) {
+  # run in CI
+  $version = $env:APPVEYOR_BUILD_VERSION -replace('\.[^.\\/]+$')
+  $version = "$version.0"
+} else {
+  # run manually
+  [xml]$spec = Get-Content docker-machine-vmwareworkstation.nuspec
+  $version = $spec.package.metadata.version
+}
+
+"TEST: Installation of docker-machine should work"
+. choco install -y docker-machine
```

Comment:
Do we need this section? should be just let the install docker-machine-vmwareworkstation pull the dependancy?